doc/source/getting_started.rst

```diff
@@ -0,0 +1,388 @@
+If you choose to use LArray in a Python script instead, you must import it::
+
+    from larray import *
+
+Create an array
+---------------
+
+Working with the LArray library mainly consists of manipulating :ref:`LArray <api-larray>` data structures.
+They represent N-dimensional labelled arrays and are composed of :ref:`axes <api-axis>`, data (numpy ndarray)
+and optionally a title. An axis contains a list of labels and may have a name (if not given, the axis is anonymous).
+
+You can create an array from scratch by supplying data, axes and optionally a title:
+
+.. ipython:: python
+   :doctest:
+
+    @suppress
+    from larray import *
+
+    data = [[2, 4], [7, 8], [11, 3]]
+    age, sex = Axis("age=0..2"), Axis("sex=M,F")
+    arr = LArray(data, [age, sex], "population by age and sex")
+    arr
+
+Axes can be also provided as a single with each axis separated by a semi-colon:
+
+.. ipython:: python
+   :doctest:
+
+    arr = LArray(data, "age=0..2;sex=M,F", "population by age and sex")
+    arr
+
+Here are the key properties for an array:
+
+.. ipython:: python
+   :doctest:
+
+    # array summary : dimensions + description of axes
+    arr.info
+
+    # number of dimensions
+    arr.ndim
+
+    # array dimensions
+    arr.shape
+
+    # number of elements
+    arr.size
+
+    # size in memory
+    arr.memory_used
+
+    # type of the data of the array
+    arr.dtype
+
+Special arrays can be generated through dedicated functions:
+
+* :py:func:`ndrange` : fills an array with increasing numbers
+* :py:func:`ndtest` : same as ndrange but with axes generated automatically (for testing)
+* :py:func:`empty` : creates an array but leaves its allocated memory unchanged (i.e., it contains “garbage”. Be careful !)
+* :py:func:`zeros` : fills an array with 0
+* :py:func:`ones` : fills an array with 1
+* :py:func:`eye` : identity matrix
+* :py:func:`full` : fills an array with a given value
+
+Save/Load an array
+------------------
+
+The LArray library offers many I/O functions to read and write arrays in various formats
+(CSV, Excel, HDF5, pickle). For example, to save an array in a HDF5 file, call the method
+:py:meth:`~LArray.to_hdf`:
+
+.. ipython:: python
+   :doctest:
+
+    arr.to_hdf('my_file.h5', 'arr')
+
+To load a saved array, call the function :py:meth:`read_hdf`:
+
+.. ipython:: python
+   :doctest:
+
+    arr = read_hdf('my_file.h5', 'arr')
+    arr
+
+Other functions are described :ref:`here <api-IO>`.
+
+Indexing
+--------
+
+You can select an element or a subset of an array by labels and/or positions.
+
+Using labels
+^^^^^^^^^^^^
+
+To select an element or a subset of an array using labels, use brackets [ ].
+Let’s start by selecting a single element:
+
+.. ipython:: python
+   :doctest:
+
+    arr = ndtest((4, 4))
+    arr
+
+    arr['a0', 'b1']
+
+    # labels can be given in an arbitrary order
+    arr['b1', 'a0']
+
+To select a row or column, specify the associated label:
```

Comment:
I fear that speaking about rows or columns as-is might reinforce the sentiment that LArray is for 2D arrays only (or better suited for those). Unsure how to fix this.